packages/vue/src/index.bundle.ts

```diff
@@ -0,0 +1,74 @@
+import { createVueEventProcessor } from './eventprocessor';
+
+export {
+  Breadcrumb,
+  Request,
+  SdkInfo,
+  Event,
+  Exception,
+  Response,
+  Severity,
+  StackFrame,
+  Stacktrace,
+  Status,
+  Thread,
+  User,
+} from '@sentry/types';
+
+export {
+  addGlobalEventProcessor,
+  addBreadcrumb,
+  captureException,
+  captureEvent,
+  captureMessage,
+  configureScope,
+  getHubFromCarrier,
+  getCurrentHub,
+  Hub,
+  Scope,
+  setContext,
+  setExtra,
+  setExtras,
+  setTag,
+  setTags,
+  setUser,
+  startTransaction,
+  Transports,
+  withScope,
+} from '@sentry/browser';
+
+export { BrowserOptions } from '@sentry/browser';
```

Comment:
Please clean these up. It's not necessary to have so many export statements for the same package.